docker/create-env.sh

```diff
@@ -24,23 +24,29 @@ function join_by { local IFS="$1"; shift; echo "$*"; }
 
 function create_env_files {
     new_files=()
-    for file in env_files/*.dist
-    do
+    function new_file {
+        file=$1
         new_file=${file%.dist}
         new_files+=( "$new_file" )
 
         if [[ ! -f "$new_file" ]]
         then
             cp $file $new_file
         fi
+    }
+
+    for file in env_files/*.dist
+    do
+        new_file $file
     done
+    new_file ../shopelectro/settings/local.py.dist
```

Comment:
The `create_env_files` function in `create-env.sh` may not create any configs. The single responsibility of this script is create env files, so let's do it elsewhere

Reply:
Also `.env` file will be contain the content of `../shopelectro/settings/local.py.dist` (see a 71 line) and it is absolutely unexpected behavior

Reply:
> Also .env file will be contain

y, i'll fix it

> The create_env_files function in create-env.sh

both django settings and env files are about env customisation in terms of our app.
In terms of linux env - not, of course. But app-sided script should work with app env, not with linux env.

I can rename script to `create-local-env`. Or can double it's code to separated script, if it looks confusing

Reply:
moved config processing to separated script